build_product

```diff
@@ -378,6 +378,7 @@ all_cmake_products=(
 	MACOS1015
 	OPENEMBEDDED
 	OPENEULER2203
+  AL2023
```

Comment:
indentation issue and also not alphabetically ordered.

Reply:
Done - Thanks!